### PR TITLE
pp/#965 fix hyperlink in line documentation

### LINKED ISSUE #965: Fix Hyperlink in Line Documentation
grafik

imho this should be a link to the subsection of standard line types below

### PR DESCRIPTION
**Pull request by pierrepetersmeier**
Resolves #965 
Should have worked so far @t-ober  :).

Bild
Should something be linked here too?

### CODE REVIEW

**Comment by t-ober** [comment]
!test

**Comment by pierrepetersmeier** [comment]
!test

File: docs/readthedocs/models/input/grid/line.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Representation of an AC line.
 | vRated    | kV      | Rated voltage                               |
 
 
-A list with some standard line types can be found here: `Standard Line Types`_
+A list with some standard line types can be found here: [Standard Line Types](#standard-line-types)_
```
Comment (sebastian-peter) [inline docs/readthedocs/models/input/grid/line.md:21]: The underscore character can be removed as well.

(You can find a preview of the changed documentation [here](https://powersystemdatamodel--980.org.readthedocs.build/en/980/models/input/grid/line.html))

**Review by sebastian-peter — CHANGES_REQUESTED** [review]
Please also do not forget to add a line to `CHANGELOG.md`. You can take the existing lines as a hint on how to do that. :)

**Review by sebastian-peter — APPROVED** [review]
Looks good, thanks! I just removed two more underscores that I just noticed.

**Pull request merged**